server/controllers/v1/tests/post.test.ts

```diff
@@ -87,13 +87,42 @@ describe('Test v1 - Post Controller', () => {
         tags
       );
 
-      // @ts-ignore
-      const postTags = await getPostTags(result.data.result.id);
+      const postTags = await getPostTags(result.data.result!.id);
       expect(postTags?.map((t) => t.text).sort()).toEqual(tags.sort());
 
       expect(result.status).toBe(200);
     });
 
+    it('should be able to use one existing tag, and one new tag', async () => {
+      const author = await makeValidUser();
+      const tags = ['csc108'];
+
+      await postController.createPost(
+        author.id,
+        'This is a new post!',
+        'This is a new post!This is a new post!',
+        'location',
+        10,
+        tags
+      );
```

Comment:
Should be able to create a tag without creating a post no?

Reply:
No? We only create a tag if someone creates a post and puts that tag in it

Reply:
I just meant you can create a tag via the model and skip the controller. Simplifies the test.

Reply:
I guess, but I also want to make sure that the tags can properly be associated to multiple posts in the junction table. Createpost will make the tag and that association for us.